test/gtest/ucs/test_type.cc

```diff
@@ -132,6 +132,8 @@ UCS_TEST_F(test_type, pack_float) {
                                   UCS_FP8_PACK(TEST_LATENCY, *it));
         ucs_assert((UCS_FP8_PRECISION < unpacked / *it) &&
                    (unpacked / *it <= 1));
+
+        EXPECT_EQ(unpacked, UCS_FP8_PACK_UNPACK(TEST_LATENCY, *it));
```

Comment:
You can just replace all existing `UCS_FP8_UNPACK(TEST_LATENCY, UCS_FP8_PACK(TEST_LATENCY, <value>))` by `UCS_FP8_PACK_UNPACK(TEST_LATENCY, <value>))`.

Reply:
Done.